.github/workflows/clippy_check.yml

```diff
@@ -7,9 +7,9 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions-rs/toolchain@v1
         with:
-            toolchain: nightly-2023-10-28
-            components: clippy
-            override: true
+          toolchain: nightly-2024-01-05
```

Comment:
channel = "nightly-2024-05-01" is used in flake

Reply:
Fixed to use the newer and the better Action.